low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -244,14 +244,20 @@ def _create_json_message(self, cpu, alert_type):
         self.fill_specific_info()
         alert_specific_info = self.specific_info
 
+        if alert_type == "fault":
+            description = "%s is missing." %cpu
```

Comment:
Am not sure on this. One CPU went missing ? Did you consult @ArchanaLimaye 
Same for fault_resolved!